jetty-start/src/main/java/org/eclipse/jetty/start/Licensing.java

```diff
@@ -85,7 +85,7 @@ public boolean acknowledgeLicenses() throws IOException
         String propBasedAckValue = System.getProperty(PROP_ACK_LICENSES);
         if (propBasedAckValue != null)
         {
-            StartLog.log("TESTING MODE", "Programmatic ACK - %s=%s", PROP_ACK_LICENSES, propBasedAckValue);
+            StartLog.info("Programmatic ACK - %s=%s", PROP_ACK_LICENSES, propBasedAckValue);
```

Comment:
"Programmatic ACK" -> "Automatic License Acknowledgment" - these are sysops or beginners, they don't know what "ACK" is.